### PR TITLE
Display Site wide links in one column under 240px

### LINKED ISSUE #551: Site wide links - Go to 1 column <= 240px
**Is your feature request related to a problem? Please describe.**
We support the tiny screen width of 240px in WS, so we should update the footer to make it nicer to read

| current | new design |
|---------|------------|
| image | image |

**Describe the solution you'd like**
At and below 240px, go to a single column design.

**Testing notes**
[Tester to complete]

Dev insight: Storybook and snaps

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #551

**Overall change:**
Add a new media query to display the footer list items in one column under `240px`.

**Code changes:**
- Import `GEL_GROUP_0_SCREEN_WIDTH_MAX` and `GEL_GROUP_1_SCREEN_WIDTH_MIN`
- Use `grid-auto-flow: row;` under `240px`;

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2541,6 +2541,7 @@
       "version": "1.2.1",
       "resolved": "https://registry.npmjs.org/@reach/router/-/router-1.2.1.tgz",
       "integrity": "sha512-kTaX08X4g27tzIFQGRukaHmNbtMYDS3LEWIS8+l6OayGIw6Oyo1HIF/JzeuR2FoF9z6oV+x/wJSVSq4v8tcUGQ==",
+      "dev": true,
```
Comment (dr3) [inline package-lock.json]: Can you reset this file please?

**Review by dr3 — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (DenisHdz) [reply package-lock.json]: Sure, commited by mistake.

**Review by dr3 — APPROVED** [review]

**Review by andrew-nowak — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged.

**Pull request merged**